pyproject.toml

```diff
@@ -26,6 +26,8 @@ sentence-transformers = {version ="^2.2.2", optional = true}
 evaluate = {version = "^0.4.0", optional = true}
 faiss-cpu = {version = "^1.7.1", optional = true}
 ipywidgets = {version = "^8.1.1", optional = true}
+presidio-analyzer = "^2.2.351"
+h5py = "^3.10.0"
```

Comment:
Did you intend to make these optional? It looks like you included the dependencies below in the "all" extra which you only need to do if these are optional dependencies.

Reply:
ok, as discussed in Slack, let's keep them as optional